### ISSUE TITLE
More functionality!

### ISSUE THREAD

**Issue by abmmhasan** [samyk/evercookie#94]
Well, been <a href="http://stackoverflow.com/questions/15966812/user-recognition-without-cookies-or-local-storage">reading on stack-overflow</a>. I got myself .......
Anyway, these thing (what missed) can be added here in this project!

**Comment by mkoryak** [comment]
good idea. The author of this plugin should read that huge SO post, parse out useful ideas and implement them!

It doesnt work that way.
